data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+			
+			`	The Free Worlds captain thanks you for offering your help. Spaceport workers load the turrets onto your ship, and the Free Worlds captain tells you to return to <origin> once you have dropped off the turrets for payment.`
+				accept
+	
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Kappa Centauri"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Kappa Centauri"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Yed Prior"
+		fleet "Large Southern Pirates"
+		fleet "Small Southern Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Yed Prior"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Pherkad"
+		fleet "Small Southern Pirates" 3
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Kappa Centauri"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Yed Prior"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Pherkad"
+		fleet "Large Free Worlds"
+	on stopover
+		dialog `The last of the defenses are quickly unloaded from your ship, put into place, and activated. One of the pirate ships attacking the system attempts to follow you down to the surface, but the fire coming from the newly installed spaceport turrets forces the pirate to turn away and flee for orbit. Time to return to <destination>.`
+	on complete
+		payment
+		payment 5000
+		dialog `The Free Worlds captain thanks you for helping to fortify the frontier worlds, and pays you <payment>.`
+
+
+
+mission "Core Blockade"
+	minor
+	name `Bad Harvest`
+	description `The pirate world of <destination> is experiencing a famine. Warlords of the planet have threatened the Syndicate to give them food, and the Syndicate has agreed to their demands. Deliver <cargo> to the world for <payment>.`
+	source
+		near "Almach" 2 4
+	destination "Deadman's Cove"
+	cargo "food" 287
+	to offer
+		random < 20
+		"combat rating" > 400
+	
+	on offer
+		conversation
+			`In the <origin> spaceport, you are approached by a Syndicate employee. By the looks of his uniform, he works for Syndicated Security. "Hello, Captain. You're a competent pilot, correct?" You nod your head. "Excellent. The Syndicate needs <cargo> transported to <destination>. The pirate world has seen a particularly bad harvest this past season, so much of the population is without sufficient food. Pirate warlords have threatened to begin raiding our freighters and frontier worlds if we don't provide them with food. It's cheaper to do what they want, so we've agreed to the deal, but we don't want to risk any of our own ships being destroyed. The Syndicate will pay you <payment> should you accept and complete this job."`
```

Comment:
The player should probably have positive rep with the Syndicate for them to offer this mission (similar conditions for the other missions is probably wise too).

Reply:
I'm sure there are plenty of missions in the game where this should be the case.

Reply:
Put rep checks on the missions that involve ships other than pirates.

Reply:
Most spawned ships are Merchant or Pirate, and there's definitely an assumption of goodwill on the player's part. A PR to add appropriate rep checks where needed should be done, but no need to wait for that PR to fix the issue here.

My thought was that the Syndicate isn't going to pay you if they're interested in killing you, so this mission doesn't make sense to be offered at that time.